src/consts.js

```diff
@@ -6,28 +6,32 @@ import rhelCs from './components/rhel-cs.svg';
 export const activeTechnologies = [
     {
         name: 'insights',
+        id: 'insights',
         url: 'insights/actions',
         icon: insights,
```

Comment:
We can now use pf icons to these, not sure if we want to use them in this PR or in followup.

Reply:
I mean I get that any XML can be embedded like this, but that might come with a penalty.

I think this embeds the XML inside of the bundle instead of serving the XML as a separate file. Right?

If that is the case we should avoid this, we have different caching rules for different content types and images, pdfs, svg etc should be cached longer than the js bundles.

I think the best argument I've seen to inline them is so that we can use CSS to style them. i.e. we can select a path and change it's color to blue.

Is that the reason we are doing this?

Reply:
Let's add it in a future update

Reply:
Let's put it in a followup PR